praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/rest/PraktikumsstellenController.java

```diff
@@ -88,6 +91,20 @@ public PraktikumsstelleDto assignNwk(@PathVariable(name = "praktikumsstellenId")
         return praktikumsstellenService.assignNwk(praktikumsstellenId, nwkId);
     }
 
+    @PreAuthorize("hasRole('ROLE_' + T(de.muenchen.oss.praktikumsplaner.security.AuthoritiesEnum).AUSBILDUNGSLEITUNG.name())")
+    @PutMapping("/ausbildung/{praktikumsstellenId}")
+    public void updateAusbildungPraktikumsstelle(@PathVariable(name = "praktikumsstellenId") UUID praktikumsstellenId,
+            @RequestBody AusbildungsPraktikumsstelleWithMeldezeitraumAndAssignedNWKDto praktikumsstelleDto) {
+        praktikumsstellenService.updateAusbildungsPraktikumsstelle(praktikumsstellenId, praktikumsstelleDto);
+    }
+
+    @PreAuthorize("hasRole('ROLE_' + T(de.muenchen.oss.praktikumsplaner.security.AuthoritiesEnum).AUSBILDUNGSLEITUNG.name())")
+    @PutMapping("/studium/{praktikumsstellenId}")
+    public void updateStudiumPraktikumsstelle(@PathVariable(name = "praktikumsstellenId") UUID praktikumsstellenId,
+            @RequestBody StudiumsPraktikumsstelleWithMeldezeitraumAndAssignedNwkDto praktikumsstelleDto) {
+        praktikumsstellenService.updateStudiumsPraktikumsstelle(praktikumsstellenId, praktikumsstelleDto);
+    }
+
```

Comment:
The Put Endpoint should be implemented like the other Put endpoints in our application.
The Request should not get the id via a parameter but rather as part of the dto.

Reply:
Proposal is not implemented as it has been agreed with the trainer.